src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -155,31 +169,50 @@ export default class DetailsBody extends React.Component<Props> {
   }
 
   renderChartHeader() {
-    const {incident} = this.props;
+    const {incident, params} = this.props;
     const alertRule = incident?.alertRule;
 
     return (
       <ChartHeader>
-        {this.friendlyIncidentType}
-
-        <ChartParameters>
-          {tct('Metric: [metric] over [window]', {
-            metric: <code>{alertRule?.aggregate ?? '...'}</code>,
-            window: (
-              <code>
-                {incident ? (
-                  <Duration seconds={incident.alertRule.timeWindow * 60} />
-                ) : (
-                  '...'
-                )}
-              </code>
-            ),
-          })}
-          {alertRule?.query &&
-            tct('Filter: [filter]', {
-              filter: <code>{alertRule.query}</code>,
+        <div>
+          {this.metricPreset?.name ?? t('Custom metric')}
+          <ChartParameters>
+            {tct('Metric: [metric] over [window]', {
+              metric: <code>{alertRule?.aggregate ?? '...'}</code>,
```

Comment:
```suggestion
              metric: <code>{alertRule?.aggregate ?? '\u2026'}</code>,
```

Ellipsis looks better than three periods.